pkg/cloudprovider/azure/block_store.go

```diff
@@ -64,6 +64,10 @@ type snapshotIdentifier struct {
 	name          string
 }
 
+func (si *snapshotIdentifier) fullName() string {
```

Comment:
Would it make more sense for this to fulfill the `Stringer` interface - change `fullName` to `String`?

Reply:
i guess it wouldn't hurt